client/cornerselling.lua

```diff
@@ -299,9 +290,8 @@ end
 -- Events
 RegisterNetEvent('qb-drugs:client:cornerselling', function()
     if CurrentCops >= config.minimumDrugSalePolice then
-        local result = lib.callback.await('qb-drugs:server:getAvailableDrugs', false)
-        if result then
-            availableDrugs = result
+        local hasDrugs = lib.callback.await('qb-drugs:server:getDrugOffer', false) ~= nil and true or false
```

Comment:
Smaller thing to convert it to a boolean regardless of the value returned
```suggestion
        local hasDrugs = not not lib.callback.await('qb-drugs:server:getDrugOffer', false)
```

Reply:
doing it that way would also work but IMO would give up a little bit on readibility, not too sure if double negation is common in Lua as i am new to it myself (I am coming from a zoo of explicit typings). regardless if you like it that way and fits your codebase please reply one more time so I can update it